Demo/AuthenticatorDemo/ViewController+WordPressAuthenticator.swift

```diff
@@ -76,5 +78,24 @@ extension ViewController {
                 navTitleTextColor: .white
             )
         )
+
+        WordPressAuthenticator.shared.delegate = self
+    }
+
+    // TODO: Need to handle new user flow
+    func newGoogleSignInFlow() {
+        Task.init {
+            do {
+                let token = try await self.googleAuthenticator.getOAuthToken()
+
+                presentAlert(
+                    title: "🎉",
+                    message: "Successfully authenticated with Google.\n\nEmail in received token: \(token.email)",
+                    onDismiss: {}
+                )
+            } catch {
+                presentAlert(title: "❌", message: error.localizedDescription, onDismiss: {})
```

Comment:
Does `presentAlert` give us any guarantees that it runs on the main thread?

Reply:
I couldn't find anything in the docs about whether the UIKit method runs on the main thread.

This is a custom method so we can definitely force it to run on the main thread. 

However, I'm not sure why we would go to "such effort" in the context of the demo app? What am I missing?

FWIW, TSan doesn't complain in the successful branch of the `do catch`.